File: yukicoder/427_テレビ/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        long HEIGHT = sc.nextInt();
        long WIDTH = sc.nextInt();

        if (WIDTH < HEIGHT) {
            System.out.println("TATE");
        } else if (HEIGHT < WIDTH){
            System.out.println("YOKO");
        }

    }

}

File: yukicoder/427_テレビ/Main.cpp
#include <iostream>

using namespace std;

int main() {

    long HEIGHT, WIDTH;
    cin >> HEIGHT >> WIDTH;

    if (WIDTH < HEIGHT) {
        cout << "TATE" << endl;
    } else if (HEIGHT < WIDTH){
        cout << "YOKO" << endl;
    }

    return 0;

}
